src/utils/type-transform/BackupFileShortRepresentation.ts

```diff
@@ -5,6 +5,7 @@ import type { ProfileRecord } from '../../database/Persona/Persona.db'
 
 export type BackupJSONFileLatestShort = [
     string, // version, should be "1"
```

Comment:
Notice: if you changed the format, you should change the version too

Reply:
Actually, the format of the backup data didn't change. It's kinda bugfix. The user is able to restore the persona with the QR code which generates before these changes.